includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -863,6 +870,15 @@ public function process_payment( $order_id ) {
 		$order = wc_get_order( $order_id );
 
 		try {
+			$fraud_prevention_service = Fraud_Prevention_Service::get_instance();
+			// phpcs:ignore WordPress.Security.NonceVerification.Missing,WordPress.Security.ValidatedSanitizedInput.MissingUnslash,WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
I've listed all 3 ignores here but I'm not sure if that's the best approach.

Looking at other similar cases where we access `$_POST` variables I don't see anything specific being done as well as phpcs:ignore statements.

Reply:
Overall this looks safe.

But we could improve the solution security:
- wp_generate_password is by default generates 12 characters long password, I'd suggest increasing the length to 16
- instead if using `===` when comparing tokens we should try using `hash_equals` (hash timing attacks prevention, when someone sends loooong values intentionally to bring the server down).